python/demo/elasticity/demo_elasticity.py

```diff
@@ -101,11 +101,11 @@ def sigma(v):
 # Define variational problem
 u = TrialFunction(V)
 v = TestFunction(V)
-a = inner(sigma(u), grad(v)) * dx
-L = inner(f, v) * dx
+a = form(inner(sigma(u), grad(v)) * dx)
+L = form(inner(f, v) * dx)
 
 # Set up boundary condition on inner surface
-bc = DirichletBC(np.array([0, 0, 0], dtype=PETSc.ScalarType),
+bc = dirichletbc(np.array([0, 0, 0], dtype=PETSc.ScalarType),
```

Comment:
Should this be `Dirichletbc` as Dirichlet is a name?

Reply:
No - we use lower case for functions and reserve upper case for classes.